src/utils.js

```diff
@@ -263,22 +264,36 @@ export const getIntersection = (extent1, extent2) => {
 
 /**
  * 
- * @param {Number} viewPortRange Size of requested viewPort in pixels
- * @param {Number} level Requested map level
  * @param {Array.<Longitude, Latitude>} coordinates Center coordinates of viewPort
- * @param {Number?} tileSizePX Size of tile in pixels. If undefined, use gridConstants.PIXEL_TILE_SIZE.
- * @param {Number} bufferCoefficient Multiply calculated visible distance by coefficient for ensure buffer around viewport.
+ * @param {Number} range Size of map in meters
+ * @param {Number} optLat Optimized range for latitude. 
```

Comment:
Better naming of parameter & description

Reply:
Selected latitude with minimized distortion.